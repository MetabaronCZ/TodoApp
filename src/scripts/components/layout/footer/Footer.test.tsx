import React from 'react';

import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import { withMockedProviders } from 'test/component';
import { Footer } from 'components/layout/footer/Footer';

describe('components/layout/Footer', () => {
  it('should render correctly', () => {
    const tree = render(withMockedProviders(<Footer />));
    expect(tree.container).toMatchSnapshot();
  });
});
