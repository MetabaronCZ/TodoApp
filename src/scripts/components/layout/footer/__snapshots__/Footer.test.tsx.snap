// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`components/layout/Footer > should render correctly 1`] = `
.c1 {
  font-family: inherit;
  font-size: inherit;
  line-height: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #22c;
}

.c1:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c1:focus {
  outline: 1px dotted #22c;
  outline-offset: -1px;
}

.c0 {
  font-family: Open Sans,Arial,sans-serif;
  color: #222;
  font-size: 14px;
  line-height: 20px;
  padding: 20px 0;
  text-align: center;
}

<div>
  <footer
    class="c0"
  >
    <div
      class=""
    >
      Copyright Â© 2023 MetabaronCZ (
      <a
        class="c1"
        href="https://github.com/MetabaronCZ"
        target="_blank"
      >
        https://github.com/MetabaronCZ
      </a>
      ). All Rights Reserved.
    </div>
  </footer>
</div>
`;
